.github/workflows/publish.yml

```
@@ -30,6 +36,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      # 2️⃣ Configuración de Node con OIDC habilitado
+      # Nota: Si tu acción local 'setup-node-deps' no soporta auth-oidc, 
+      # podrías necesitar usar actions/setup-node@v4 directamente aquí.
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20' # Ajusta según tu .nvmrc si es necesario
+          registry-url: 'https://registry.npmjs.org/'
+          auth-oidc: true
```

actionlint (1.7.12)

[error] input "auth-oidc" is not defined in action "actions/setup-node@v4". available inputs are "always-auth", "architecture", "cache", "cache-dependency-path", "check-latest", "mirror", "mirror-token", "node-version", "node-version-file", "registry-url", "scope", "token"

(action)